lib/configs/flat/base.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @fileoverview flat config - unlike eslintrc configs, it's not auto generated.
```

Comment:
Hmm, why are flat configs no autogenerated? I think they have to be autogenerated, so we don't forget to add rules to all relevant configs.

Reply:
not familiar with the generating logic, it seems hard to share categories for eslintrc and flat configs?

https://github.com/vuejs/eslint-plugin-vue/blob/024742478ca726d6b649b5102379e1ed34e72889/tools/lib/categories.js#L10-L38

Reply:
you can always add it back when having consistent naming with eslintrc.(maybe next major version). thoughts?

Reply:
Hmm, the logic is not that complicated:

```js
function getFlatConfigName(categoryTitle) {
  if (categoryTitle === 'base') {
    return 'flat/base'
  }
  if (categoryTitle.startsWith('vue3-') {
    return `flat/${categoryTitle.replace('vue3-', '')}`
  }
  return `flat/vue2-${categoryTitle}`
}
```

I'd prefer having it autogenerated right now.

Reply:
well, I will give it a try.

Reply:
added it in ca03026e2b1078791e7bb5f9e77207dd74c458c8.